### ISSUE TITLE
Update to Google Java Format 1.1

### ISSUE THREAD

**Issue by tbroyer** [sherter/google-java-format-gradle-plugin#12]
Using:
```gradle
googleJavaFormat {
  toolVersion '1.1'
}
```
produces:
```
$ ./gradlew gJF 
:googleJavaFormat
Version 0.3.2 of google-java-format-gradle-plugin is not tested against version 1.1 of google-java-format. This should not be a problem if the task is executed without failures.
:googleJavaFormat FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':googleJavaFormat'.
> java.lang.ClassNotFoundException: com.google.googlejavaformat.java.JavaFormatterOptions$JavadocFormatter

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.

BUILD FAILED

Total time: 0.59 secs
```

**Comment by sherter** [comment]
I started working on it, just didn't have the time to finish it... hopefully this weekend! ;)

**Comment by sherter** [comment]
Fixed in v0.4.1.

**Comment by tbroyer** [comment]
Thanks, that was fast!
